robot:
  hardware_interface:
    loop_hz: 50 # hz
    joints:
      - base_controller

    simulate_joints: true

  rosserial:
    port: "/dev/ttyS3"
    baud: "115200"
